case-lib/lib.sh

```diff
@@ -40,6 +40,41 @@ if [ ! "$SOFCARD" ]; then
 		awk '/sof-[a-z]/ && $1 ~ /^[0-9]+$/ { $1=$1; print $1; exit 0;}')
 fi
 
+# Arguments:
+#
+#   - poll interval in secs
+#   - timeout in secs, rounded up to the next interval
+#   - command and arguments
+#
+poll_wait_for()
+{
+    test $# -ge 3 ||
+        die "poll_wait_for() invoked with $# arguments"
```

Comment:
if the user use it wrong, maybe we should inform the correct usage?

Reply:
We could but this function is not meant to be used outside sof-test, so the only user using it wrong is either you or me.

More seriously, the only user using it wrong is someone writing sof-test code who can just open the file and look at the header above = what they should have done already before even starting to use the function.